check-new-release

```diff
@@ -40,15 +40,22 @@ for app_config in "."/*.cfg; do
         exit 1
     fi
 
-    upstream_version=$(echo "$tags" | jq -r '.[0].name' | sed -E 's/v//g')
+    NEWEST_TAG_INDEX=0
+    STABLE_VERSION_FOUND=false
 
-    if [ -z "$upstream_version" ]; then
-        echo "Unable to retrieve upstream version"
-        exit 1
-    elif [[ $upstream_version == *"RC"* ]] || [[ $upstream_version == *"rc"* ]] || [[ $upstream_version == *"beta"* ]]; then
-        echo "Latest upstream version is RC or beta: $upstream_version. Skipping"
-        continue
-    fi
+    while ! $STABLE_VERSION_FOUND; do
+        upstream_version=$(echo "$tags" | jq -r ".[$NEWEST_TAG_INDEX].name" | sed -E 's/v//g')
+
+        if [ -z "$upstream_version" ]; then
+            echo "Unable to retrieve upstream version"
+            exit 1
+        elif [[ $upstream_version == *"RC"* ]] || [[ $upstream_version == *"rc"* ]] || [[ $upstream_version == *"beta"* ]]; then
```

Comment:
I'd add also the `alpha` tag

Also, enable the case-insensitve match so you do not have to check multiple times for the same string (https://stackoverflow.com/questions/1728683/case-insensitive-comparison-of-strings-in-shell-script).

Reply:
Done. Thanks for the suggestion